UnitySetup/UnitySetup.psm1

```diff
@@ -2254,8 +2246,12 @@ Would you like to continue? (Default: $($createPAT))"
             }
             Connect-AzAccount @connectArgs  | Out-Null
         }
-        $AZTokenRequest = Get-AzAccessToken -ResourceType Arm
-        $headers = @{ Authorization = "Bearer $($AZTokenRequest.Token)" }
+
+        $AZToken = $null
```

Comment:
Do we need to set this to $null? We seem to always set the value on line 2252, so I believe this can be removed.

Reply:
Good call - no need for that line anymore.